packages/dropdowns/src/examples/menu.md

```diff
@@ -0,0 +1,229 @@
+## Basic Menu Layout
+
+```jsx
+const { Button } = require('@zendeskgarden/react-buttons/src');
+const { IconButton, Icon } = require('@zendeskgarden/react-buttons/src');
+const SettingsIcon = require('@zendeskgarden/svg-icons/src/16/gear-stroke.svg').default;
```

Comment:
Any reason `require` is used throughout?

Reply:
This is a side-effect of styleguidist. Just specific to our examples.